build-system/pr-check/remote-tests.js

```diff
@@ -44,10 +45,11 @@ function main() {
 
   if (!isTravisPullRequestBuild()) {
     startSauceConnect(FILENAME);
-    downloadBuildOutput(FILENAME);
 
+    downloadBuildOutput(FILENAME);
```

Comment:
This is a good opportunity to run unit tests as well on a closure-compiled (`gulp dist`) binary. I suggest removing the line above, and move `downloadDistOutput(FILENAME)` to the top.

Reply:
Done!